tests/src/Common/test_dependencies/project.json

```diff
@@ -2,6 +2,7 @@
   "dependencies": {
     "Microsoft.NETCore.Platforms": "2.0.0-beta-25114-01",
     "Microsoft.Private.CoreFx.NETCoreApp": "4.4.0-beta-25114-01",
+    "System.Memory": "4.4.0-beta-25114-01",
```

Comment:
I am not sure exactly why this is needed. Isn't Span<T> also already defined in mscorlib?